service/terraform-plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/terraform/manager/TerraformLogProcessingService.java

```diff
@@ -28,6 +29,8 @@ public class TerraformLogProcessingService {
     public static final String VALUE_TYPE_APPLY_COMPLETE = "apply_complete";
     public static final String VALUE_TYPE_APPLY_ERRORED = "apply_errored";
     public static final String KEY_MESSAGE = "@message";
```

Comment:
We standardized the log format. Every log will have at least a message, level and timestamp.